sympy/logic/boolalg.py

```diff
@@ -994,6 +1061,40 @@ def _distribute(info):
         return info[0]
 
 
+def to_anf(expr, complete=True):
+    """
+    Converts expr to Algebraic Normal Form (ANF).
+
+    ANF is a canonical normal form, which means that two
+    equivalent formulas will convert to the same ANF.
+
+    A logical expression is in ANF if it has the form
+    ((A & B & ...) ^ (B & C ...) ^ B ^  True ^ ...),
+    i.e. it is purely true, purely false or exclusive disjunction.
+    The exclusive disjunction can only contain true, variables or
+    conjunction of variables. No negations are permitted.
+
+    If ``complete`` is false, arguments of the boolean
+    expression considered as variables.
```

Comment:
Yes, exactly. I didn't know how to name it properly.

I used the conversions:
- Or(x<sub>1</sub> , x<sub>2</sub>) = Xor(x<sub>1</sub> , x<sub>2</sub>, And(x<sub>1</sub> , x<sub>2</sub>));
- Or(x<sub>1</sub> , x<sub>2</sub>, ... , x<sub>n</sub>) = Xor(x<sub>1</sub>, Or(x<sub>2</sub>, ... , x<sub>n</sub>), And(x<sub>1</sub>, Or(x<sub>2</sub>, ... , x<sub>n</sub>)));
- Not(x) = Xor(x, True);
- Implies(a, b) = Xor(a, True, And(a, b));
- Equivalent(x<sub>1</sub> , x<sub>2</sub>) = Or(And(x<sub>1</sub> , x<sub>2</sub>), And(~x<sub>1</sub> , ~x<sub>2</sub>)) = Xor(x<sub>1</sub> , x<sub>2</sub>, True);
- Equivalent(x<sub>1</sub> , x<sub>2</sub>, ... , x<sub>n</sub>) = Or(And(x<sub>1</sub> , x<sub>2</sub>, ... , x<sub>n</sub>), And(~x<sub>1</sub> , ~x<sub>2</sub>, ... , ~x<sub>n</sub>)) = Xor(x<sub>1</sub> , x<sub>2</sub>, ... , x<sub>n</sub>, True, ....)